decision_tree.py

```diff
@@ -0,0 +1,60 @@
+import time
+from time import sleep
+from variables import *
+
+start()
+decision1 = input(ques1)
+sleep(1)
+
+if decision1 == ("1"):
+    snooze()
+    decision2 = input(ques2)
+    if decision2 == ("1"):
+        brush_teeth()
+        decision3 = input(ques3)
+        if decision3 == ("1"):
+            go_school()
+            decision4 = input(ques4)
+            if decision4 ==("1"):
+                run_home()
+            if decision4 == ("2"):
```

Comment:
Could be elif here, so that you don't have to check decision4 twice.
```suggestion
            elif decision4 == ("2"):
```